contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6159,11 +6162,40 @@ transformSelectIntoStmt(CreateTableAsStmt *stmt)
 			}
 			else
 			{
+				AlterTableStmt *newstmt;
+				AlterTableCmd *cmd;
+				char* original_name = NULL;
+				newstmt = NULL;
+
+				if (tle->resname != NULL)
+				{
+					original_name = tle->resname;
+					tle->resname = downcase_identifier(tle->resname, strlen(tle->resname), false, false);
```

Comment:
Can this `tle-resname` be truncated if the name is larger than 64 chars? If yes then we are going to store truncated original name which is not desirable. Let's check it once, otherwise we will need to fetch the original_name from query string same as how we generally do it.